### ISSUE TITLE
Setting width and height to fallback img is restricting size regardless of source used

### ISSUE THREAD

**Issue by tvanantwerp** [11ty/eleventy-img#66]
I seem to be having a problem with the setting of `width` and `height` attributes for the fallback `img` tag created inside the `picture` element when using `generateHTML`. It's correctly generating other sizes and creating the `source` elements. But the image always renders as the smallest size. I know it's actually choosing a correct source because the `currentSrc` DOM property is right. When I manually remove `width` and `height` in devtools, it immediately renders to the correct size for the given width.

From looking at the source, it doesn't appear like there's an option to not generate `width` and `height` for the `img` tag when using `generateHTML` instead of writing a custom function to return HTML. While reading up on using `picture`, I couldn't find any examples where the included `img` tag had `width` or `height` set, so I'm not even sure if that should be a default.

Am I missing something? Surely this would be an issue for others besides myself if the default behavior is to always restrict image size to the smallest generated size.

Example of it rendering the smallest size.
![image](https://user-images.githubusercontent.com/2487968/107454885-d2cef680-6b1b-11eb-9e03-2ea5b80b0de7.png)

Shows correctly after manual edits to remove `width` and `height`.
![image](https://user-images.githubusercontent.com/2487968/107454865-ccd91580-6b1b-11eb-9714-3346a904653d.png)

The image generating shortcode I am using:
```javascript
async function imageShortcode(src, alt, sizes=[400, 800, 1200]) {
  let metadata = await Image(src, {
    widths: sizes,
    formats: ["avif", "webp", "jpeg"],
    outputDir: '_site/img',
    filenameFormat: (id, src, width, format) => {
      const filename = src.split('/').slice(-1)[0].split('.')[0];
      if (width) {
        return `${filename}-${id}-${width}.${format}`
      }
      return `${filename}-${id}.${format}`
    }
  });

  let imageAttributes = {
    alt,
    sizes: sizes.reverse().map(
      (size, i) => {
        return i === sizes.length - 1 ? `${size}px` : `(min-width: ${size}px) ${size}px`
      }
    ),
    loading: "lazy",
    decoding: "async",
  };

  // You bet we throw an error on missing alt in `imageAttributes` (alt="" works okay)
  return Image.generateHTML(metadata, imageAttributes, {
    whitespaceMode: "inline"
  });
}
```

**Comment by zachleat** [comment]
Yeah, I totally agree with you. I think this is a duplicate of #63. I did write some background about my change of heart about those attribute defaults here: https://www.zachleat.com/web/fluid-images/

TL;DR The current method works as expected with `width: 100%` CSS but using the largest source to feed width/height works better with `max-width: 100%` approaches (and good with `width: 100%` too).

I am going to close this as a dupe though—follow along at #63!
